src/towncrier/test/test_check.py

```diff
@@ -166,6 +166,71 @@ def test_fragment_missing(self):
                 result.output.endswith("No new newsfragments found on this branch.\n")
             )
 
+    def test_fragment_exists_but_not_in_check(self):
```

Comment:
Can you please add a paragraph describing the scope and the end user functionality tested here ?

Same comment as for `def test_fragment_exists_but_in_check(self):`

I know that as part of this PR, the tests make sense.

but if someone reads these tests one year into the future, and without being familiar with the check feature, it might not be that obvious

No need to be verbose. Just the important part

See https://jml.io/test-docstrings/


--------


We don't have tests for the testing code.

So the test docstring is the only input that can be used to manualy validate / check that the test code is valid.